sdk/go/common/util/executable/executable.go

```diff
@@ -27,12 +28,38 @@ import (
 
 const unableToFindProgramTemplate = "unable to find program: %s"
 
+type NotFoundError struct {
```

Comment:
Doesn't look like anything inits this?

Reply:
Yeah :facepalm: added now.

I've added a test for windows now as well, so we should actually see this work, since I don't have a windows machine available.